SharpieBinder/CxxBinder.cs

```diff
@@ -1485,6 +1485,7 @@ public override void VisitCXXMethodDecl(CXXMethodDecl decl, VisitKind visitKind)
 						{"const class Urho3D::Matrix4 &", "Matrix4"},
 						{"const class Urho3D::Matrix3x4 &", "Matrix3x4"},
 						{"int", "int"},
+						{"int&", "int"},
```

Comment:
Maybe this should be a ref int? Otherwise value changes are not marshalled.

Reply:
That's what i thought but i wasn't sure, should i add it ?

Reply:
The reason I ask is that if you get an int& from native, you might marshal the wrong value in the end, a stack pointer rather than the real value.

Reply:
So `ref int` should be the marshalled type.